subprojects/docs/src/docs/userguide/tutorial/part4_gradle_plugins.adoc

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[part4_gradle_plugins]]
+= Part 4: Apply Gradle Plugins and Distribute your App
+
+Learn about how Plugins extend Gradle's capabilities and use a popular plugin to distribute your app.
+
+****
+**In this section you will:**
+
+- Apply a plugin
+- Use the plugin
+- Learn about distribution
+- Explore other plugins
+****
+
+[[part4_begin]]
+== Before you begin
+
+1. Complete <<part3_gradle_dep_man#part3_begin,part 3>>.
+
+== Step 1. Apply the Plugin
+A plugin is simply a set of tasks, available as packaged code that use the Gradle API to provide additional functionality and extend Gradle core.
+Applying a plugin to a project means that it allows the plugin to extend the project’s capabilities.
```

Comment:
```suggestion
Applying a plugin to a project executes code that can create tasks, configure properties, and otherwise extend the project's capabilities.
```